src/main/java/org/synyx/urlaubsverwaltung/core/account/service/VacationDaysService.java

```diff
@@ -61,7 +61,7 @@ public BigDecimal calculateTotalLeftVacationDays(Account account) {
         VacationDaysLeft vacationDaysLeft = getVacationDaysLeft(account);
 
         // it's before April - the left remaining vacation days must be used
-        if (DateUtil.isBeforeApril(nowService.now()) && account.getYear() == nowService.currentYear()) {
+        if (DateUtil.isBeforeApril(nowService.now(), account.getYear())) {
             return vacationDaysLeft.getVacationDays().add(vacationDaysLeft.getRemainingVacationDays());
```

Comment:
> dieser PR betrifft nur die Anzeige des Resturlaubs, Probleme bei der Berechnung desgleichen bleiben weiterhin bestehen

Okay, das war gelogen. Diese Änderung hier in VacationDayService beeinflusst auch die Berechnung des Resturlaubs. Sowohl die alte als auch die neue Version sind unvollständig. Anstelle des aktuellen Datums muss zur Berechnung der gewünschte Zeitpunkt per Parameter übergeben werden: Wenn jemand Urlaub im Februar nächsten Jahres beantragt, darf der Resturlaub genutzt werden, im Juli nächstens Jahres aber nicht (beides unabhängig vom Datum des Urlaubsantrags, i.e. "heute").
Fortsetzung folgt..